tests/tracee_test.go

```diff
@@ -46,58 +44,6 @@ func runCommand(t *testing.T, cmd string, args ...string) string {
 	return string(output)
 }
 
-// TestLaunchTracee tests the basic sanity workflow of running tracee
-// and detecting an attack by simulating a signature trigger
-func TestLaunchTracee(t *testing.T) {
```

Comment:
I wasn't sure if we want to keep TestWebhookIntegration. @simar7  what was the idea behind it? You wanted to run it on each PR, nightly, weekly, before release? I think it's not very useful if we don't run such tests.

Reply:
The idea was to test webhook :) but more importantly it was giving us a better way to test output by using an httptestserver. 

Maybe we have better tests now and it needs to go.